test/CCR/CCRutils/CCRConfig.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity ^0.8.0;
+
+import { IWormhole } from "wormhole-solidity-sdk/interfaces/IWormhole.sol";
+import "contracts/interfaces/wormhole/IWormholeTransceiver.sol";
+
+contract CCRConfig {
+    // Standard Structure defined to store Source chain and destination chain address.
+    struct SourceConfig {
+        IWormhole wormhole;
+        IWormholeTransceiver wormholeTransceiverChain1;
+        address NTT_MANAGER;
+        address TRANSCEIVER;
+        address WORMHOLE_RELAYER_SOURCE;
+        address PUSH_NTT_SOURCE;
+        address PushHolder;
+        uint16 SourceChainId;
+        string rpc;
+    }
+
+    struct DestConfig {
+        IWormhole wormhole;
+        IWormholeTransceiver wormholeTransceiverChain2;
+        address NTT_MANAGER;
+        address WORMHOLE_RELAYER_DEST;
+        address PUSH_NTT_DEST;
+        uint16 DestChainId;
+        string rpc;
+    }
+
+    //For Arbutrum Sepolia as a source chain
+    SourceConfig ArbSepolia = SourceConfig(
```

Comment:
My idea with having two config structs, source, and destination, and then naming based on chain name was to provide a way for adding more chains in the future if needed. 

One thing we can do is to name like this: `ArbSepoliaSource` ,`EthSepoliaDest` , so that in the future we can add more chains like `PolyMumbaiSource`, `BscTestnetSource`. 

Wdyt?

Reply:
Hmm, I still think not putting chain name in the config struct is the way
Pointers:
1. I don't think we'll ever need to test multiple chain ccr coz it's one way (Comm -> Core), similar to how we test comm contract to core contract interactions (and comms are on multiple chains)
2. If we use ArbSepolia in every test or any chain name, then to test a different chain source contracts for ccr, it'd be difficult (considering we require testing for every source chain setup)
3. Even if we require multiple source chain setup testing, then we can name them SourceChain1/SourceChainOne/SourceChainPrimary or something like this to not constraint to a specific chain config and generalise it

Reply:
The test is not only for testnets, I think we will also test on the mainnet fork before actually testing on mainnet. 

It won't be difficult to switch between chains if we put a storage variable of what chain we are testing. We can just change that variable and then all of the test files will use those particular chains. 

Also using the chain name helps us to figure out the addresses, in case we want to have a look at block explorer or quickly simulate transactions on tenderly. 

So here is what I propose: 

Let's keep the chain names because the difference between Source and destination is already there with two different Structs. 

And put two variables in the helper file, `SourceChain` and `DestChain`, so that whenever we wanna test on another chain we only update these two variables. 

This will help us test via all chains, like  Arb to ETH or Polygon to ETH (mainnet and testnet both).

Reply:
Added some changes @0xNilesh 

Now when we add more chains, we will only update this [variable](https://github.com/push-protocol/push-smart-contracts/blob/734a4c4e3c6e398865b70eee6cd339775bf95786/test/CCR/CCRutils/Helper.sol#L17).

Reply:
yeah, better now 
thanks @Zartaj0